release-notes.md

```diff
@@ -1,5 +1,12 @@
 # Rigel-18.14.0
 
+## Add groupUse list permission for game service partner ##
```

Comment:
typo - groupUser
also makes more sense to put this section under the add partner -16 section